src/LinearMaps.jl

```diff
@@ -293,7 +294,9 @@ For the function-based constructor, there is one more keyword argument:
     The default value is guessed by looking at the number of arguments of the first
     occurrence of `f` in the method table.
 """
-LinearMap(A::MapOrVecOrMat; kwargs...) = WrappedMap(A; kwargs...)
+LinearMap(A::MapOrVecOrMat; getind=nothing, kwargs...) = _LinearMap(getind, A; kwargs...)
```

Comment:
```suggestion
LinearMap(A::MapOrVecOrMat; getind=Base.getindex, kwargs...) = _LinearMap(getind, A; kwargs...)
```
For a `WrappedMap` could the default `getind` simply be `Base.getindex` because presumably that is the expected behavior for a matrix.